src/utils/args.rs

```diff
@@ -4,7 +4,7 @@ use clap::{Arg, ArgAction, Command};
 
 fn validate_org(v: &str) -> Result<String, String> {
     if v.contains('/') || v == "." || v == ".." || v.contains(' ') {
-        Err("Invalid value for organization. Use the URL slug and not the name!".to_string())
+        Err("Invalid value for organization. Use the URL slug or the ID and not the name!".to_string())
```

Comment:
```suggestion
        Err("Invalid value for organization. Use the URL slug or the ID, not the name!".to_string())
```

Think this would read more clearly